contracts/contracts/strategies/FrxEthRedeemStrategy.sol

```diff
@@ -0,0 +1,241 @@
+        // solhint-disable-next-line no-unused-vars
+        uint256 _amount
+    ) external override onlyVault nonReentrant {
+        // Does nothing - all redeems need to be called manually by the
+        // strategist via redeemTickets
+        require(false, "use redeemTickets() instead");
+    }
+
+    /**
+     * @notice Redeem specific tickets from the Queue.
+     * Called by the strategist.
+     * @param _nftIds Array of NFT IDs to redeem
+     */
+    function redeemTickets(uint256[] memory _nftIds, uint256 expectedAmount)
+        external
+        nonReentrant
+    {
+        require(
+            msg.sender == IVault(vaultAddress).strategistAddr(),
+            "Caller is not the Strategist"
+        );
+        uint256 startingBalance = payable(address(this)).balance;
+        for (uint256 i = 0; i < _nftIds.length; i++) {
+            uint256 nftId = _nftIds[i];
+            redemptionQueue.burnRedemptionTicketNft(
+                nftId,
+                payable(address(this))
+            );
+            emit RedeemNFTBurned(nftId);
+        }
+
+        uint256 currentBalance = payable(address(this)).balance;
+        uint256 redeemedAmount = currentBalance - startingBalance;
+        require(
+            expectedAmount == redeemedAmount,
+            "Redeemed amount does not match expected amount"
+        );
+        outstandingRedeems -= redeemedAmount;
+        weth.deposit{ value: currentBalance }();
+        // slither-disable-next-line unchecked-transfer
+        weth.transfer(vaultAddress, currentBalance);
+    }
+
+    function _abstractSetPToken(address _asset, address _pToken)
+        internal
+        override
+    {}
+
+    /**
+     * @notice Withdraw all assets from this strategy, and transfer to the Vault.
+     * In correct operation, this strategy should never hold any assets.
+     */
+    function withdrawAll() external override onlyVaultOrGovernor nonReentrant {
+        if (payable(address(this)).balance > 0) {
+            weth.deposit{ value: payable(address(this)).balance }();
+        }
+        uint256 wethBalance = weth.balanceOf(address(this));
+        if (wethBalance > 0) {
+            // slither-disable-next-line unchecked-transfer
+            weth.transfer(vaultAddress, wethBalance);
+        }
+        uint256 fraxEthBalance = frxETH.balanceOf(address(this));
+        if (fraxEthBalance > 0) {
+            // slither-disable-next-line unchecked-transfer
+            frxETH.transfer(vaultAddress, fraxEthBalance);
+        }
+    }
+
+    /**
+     * @notice Returns the amount of queued FraxEth that will be returned as WETH.
+     * We return this as a WETH asset, since that is what it will eventually be returned as.
+     * We only return the outstandingRedeems, because the contract itself should never hold any funds.
+     * @param _asset      Address of the asset
+     * @return balance    Total value of the asset in the platform
+     */
+    function checkBalance(address _asset)
```

Comment:
nit: For all other strategies, `checkBalance` reverts if asset isn't supported. This is a temporary strategy, so it shouldn't matter though

Reply:
Changed to match